### PR TITLE
Update containerized install guide to incorporate gateway

### PR DESCRIPTION
**Pull request by michellemacrh**
Update containerized installer product docs to incorporate Gateway

https://issues.redhat.com/browse/AAP-18760

### CODE REVIEW

File: downstream/assemblies/platform/assembly-aap-containerized-installation.adoc [hunk 1]
```diff
@@ -15,28 +15,27 @@ endif::[]
...
-* SSH keys are only required when installing on remote hosts. If doing a self contained local VM based installation, you can use *ansible_connection: local* as per the example which does not require SSH.
-* Internet access from the RHEL host if using the default online installation method.
+* A RHEL 9.2 based host. Minimal operating system (OS) base install is recommended.
+* A non-root user for the {RHEL} host, with sudo or other Ansible supported privilege escalation (sudo recommended). This user is responsible for the installation of containerized {PlatformNameShort}.
+* It is recommended that you set up an *SSH public key authentication* for the non-root user. For guidelines on setting up an SSH public key authentication for the non-root user, see link:https://access.redhat.com/solutions/4110681[How to configure SSH public key authentication for passwordless login].
+* SSH keys are only required when installing on remote hosts. If doing a self contained local VM based installation, you can use *ansible_connection: local* as in the example that does not require SSH.
```
Comment (ariordan-redhat) [inline downstream/assemblies/platform/assembly-aap-containerized-installation.adoc:27]:
```suggestion
* SSH keys are only required when installing on remote hosts. If doing a self contained local VM based installation, you can use *ansible_connection: local*.
```

File: downstream/modules/platform/proc-downloading-containerized-aap.adoc [hunk 2]
```diff
@@ -8,23 +8,23 @@
...
+. Download the latest installer .tar file from link:{PlatformDownloadUrl}[access.redhat.com]. This can be done directly within the {RHEL} host, which saves time.
 
-. If you have downloaded the tarball and optional manifest zip file onto your laptop, copy them onto your RHEL host.
+. If you have downloaded the .tar file and optional manifest .zip file onto your laptop, copy them onto your {RHEL} host.
 +
-Decide where you would like the installer to reside on the filesystem. Installation related files will be created under this location and require at least 10Gb for the initial installation.
+Decide where you want the installation program to reside on the filesystem. Installation related files will be created under this location and require at least 10Gb for the initial installation.
 +
-. Unpack the installer tarball into your installation directory, and cd into the unpacked directory. 
+. Unpack the installation program .tar file into your installation directory, and cd into the unpacked directory.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-downloading-containerized-aap.adoc:17]:
```suggestion
. Unpack the installation program .tar file into your installation directory, and navigate to the unpacked directory. 
```

File: downstream/modules/platform/ref-using-custom-tls-certificates.adoc [hunk 3]
```diff
@@ -8,27 +8,33 @@
 
 [role="_abstract"]
 
-By default, the installer generates TLS certificates and keys for all services which are signed by a custom Certificate Authority (CA). You can provide a custom TLS certificate/key for each service. If that certificate is signed by a custom CA, you must provide the CA TLS certificate and key.
+By default, the installation program generates TLS certificates and keys for all services that are signed by a custom Certificate Authority (CA). You can provide a custom TLS certificate and key for each service. If that certificate is signed by a custom CA, you must provide the CA TLS certificate and key.
```
[the remark this answers is not in the document]
Comment (michellemacrh) [reply downstream/modules/platform/ref-using-custom-tls-certificates.adoc:11]: I think the style guide recommends the term "installation program" so I was going with that 🤔 I can change it back if installer is the preferred term 👍

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**